Description: Bug fix: Remove schema name from CREATE INDEX
 Indexes are always created in the same schema as the table.
Author: Jochen Topf <jochen@topf.org>
Origin: https://github.com/openstreetmap/osm2pgsql/issues/1436
Bug: https://github.com/openstreetmap/osm2pgsql/issues/1447

--- a/src/middle-pgsql.cpp
+++ b/src/middle-pgsql.cpp
@@ -714,9 +714,9 @@ static table_sql sql_for_ways(bool has_b
             "  SELECT ARRAY(SELECT DISTINCT"
             "    unnest($1) >> {way_node_index_id_shift})\n"
             "$$ LANGUAGE SQL IMMUTABLE;\n"
-            "CREATE INDEX {schema}{prefix}_ways_nodes_bucket_idx"
-            "  ON {schema}{prefix}_ways"
-            "  USING GIN ({schema}{prefix}_index_bucket(nodes))"
+            "CREATE INDEX \"{prefix}_ways_nodes_bucket_idx\""
+            "  ON {schema}\"{prefix}_ways\""
+            "  USING GIN ({schema}\"{prefix}_index_bucket\"(nodes))"
             "  WITH (fastupdate = off) {index_tablespace};\n";
     }
 
